### DISCUSSION TITLE
How can multiple requests use the same QuicStreamChannel

### DISCUSSION THREAD

**Discussion by cosmoswei** [netty/netty-incubator-codec-http3#292] [Q&A]
How to reuse the StreamChannel Stream? I wanted to make multiple requests using Http3ClientExample, but now every time I make a request, I re-create a StreamChannel using Http3#newRequestStream.

**Comment by normanmaurer** [comment]
Thats exactly how it must be... in HTTP/3 each request / response cycle is a new stream.
